lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -38,6 +38,7 @@
             - >
               The available states are: C(present) (default) and C(absent). If the screen already exists, and the state is not C(absent), the screen
               will be updated as needed.
+            - You can sort hosts aplabetically, set C(sort) option to C(true)
```

Comment:
Options should be marked with `I()` instead of `C()` as it is reserved for option values. See [documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation).

But, lets ignore it for now, beacause I believe that this module needs some refactoring as a whole.